orchestration/dagster_orchestration/hca_manage/manage.py

```diff
@@ -62,8 +66,12 @@ def __post_init__(self):
     def bigquery_client(self) -> bigquery.client.Client:
         return bigquery.Client(project=self.project)
 
+    @cached_property
+    def gcp_creds(self) -> google.auth.credentials.Credentials:
```

Comment:
❓ What is using this? I can't find a reference to it

Reply:
Updated to make methods use it internally.